### ISSUE TITLE
tango device as argument

### ISSUE THREAD

**Issue by EmCeBeh** [MBI-Div-B/taurus-powermeter#1]
could you please make the tango device for this gui configurable as argument

**Comment by dschick** [comment]
The code for getting the arguments can look like this
```python
from taurus.core.util.argparse import get_taurus_parser

parser = get_taurus_parser()
parser.set_usage("python powermeter.py domain/family/member")
parser.set_description("Taurus QtGUI for powermeter applications")

app = TaurusApplication(cmd_line_parser=parser, app_name='Powermeter')

args = app.get_command_line_args()
```

**Comment by tox1c90** [comment]
solved by pr #2
